client/src/app/kommissions/view/kommission-view.component.html

```diff
@@ -16,7 +16,7 @@ <h1>{{ kommission?.name }}</h1>
           <h2>Effectif</h2>
         </section>
       <p>
-        Les(le/la) barmen(an/aid) actifs de la kommission sont: <span *ngFor="let barman of getBarmen(true)">{{ barman.nickname }}</span>
+        Les barmen actifs de la kommission sont: <span *ngFor="let barman of getBarmen(true)">{{ barman.nickname }}</span>
```

Comment:
For now, nicknames are splited by spaces. Can we add a comma or something more visible?